### ISSUE TITLE
Allow computing biomarkers over regions instead of over channels

### ISSUE THREAD

**Issue by ssimpraga** [NeurophysiologicalBiomarkerToolbox/NBT#149]
Particularly because of connectivity biomarkers (doesn't make sense to make an average of PLI median over a certain region)...

**Comment by simon-shlomo** [comment]
How would you do this? Average the signals in a region and then calculate e.g. PLI? Would that be a valid approach?

**Comment by ssimpraga** [comment]
Yes, I was thinking of doing it that way... You don't think it would be a valid way to do it?

**Comment by simon-shlomo** [comment]
It might be valid to average the amplitude envelope and phase values.. if you average oscillations you might risk they average out, depending on how large a region you average over, and if you have dipole in the region.
